integration-tests/tests/rpc.rs

```diff
@@ -13,6 +13,20 @@ lazy_static! {
     pub static ref GEN_DATA: GenDataOutput = GenDataOutput::load();
 }
 
+#[tokio::test]
+async fn test_get_chain_id() {
+    let cli = get_client();
```

Comment:
There's a fantastic crate I discovered some time ago which will make your life easier since you always need to do the same setup for each test.

See: https://github.com/markhildreth/test-context

Reply:
What would you add in the setup step?  I can only think of the line `let cli = get_client();`. I think for a single line it's fine to keep it like this, but would take it into account in the future!

Reply:
Ye, maybe is premature now. 
But I expect in the future the possibility of the need. We can avoid it for now, it's fine! :)